docs/stencil-docs/localization/localization-tutorial.md

```diff
@@ -0,0 +1,98 @@
+# Localization Tutorial
+<div class="otp" id="no-index">
+
+### On this page
+- [Getting started](#getting-started)
+- [Creating translation keys](#creating-new-translation-keys)
+- [Update browser settings](#updating-browser-settings)
+- [Resources](#resources)
+
+</div>
+
+You have the option of localizing your theme for desired target languages. This tutorial describes the case of localizing a storefront for multiple languages. The translated values are displayed depending on the language selected in your browser. By the end of this tutorial, you should be familiar enough to localize headers, footers, buttons, payment methods, or any part of your theme.
+
+This article assumes you have familiarity with the following concepts:
+
+* [Installing Stencil CLI](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/installing-stencil)
+* [Creating a Stencil CLI Token](https://support.bigcommerce.com/s/article/Store-API-Accounts)
+* [Downloading and Uploading Custom Themes](https://support.bigcommerce.com/s/article/Stencil-Themes#download-upload)
+* [Serving a live preview](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/live-previewing-a-theme#serving-a-live-preview)
+
+### Prerequisites
+
+* For this tutorial, you will need to use Stencil CLI and use [Browsersyn](https://github.com/bigcommerce/browser-sync) to serve up a live preview of a theme in development.
+
+## Getting started
+1. Within the Stencil theme, go to the top-level `/lang` subdirectory. You will save your language files here. Each language requires its own JSON file. 
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+<!-- theme:  -->
+> **NOTE:** The `/lang` subdirectory includes the `/en.json` file. Both files must be present for a Stencil theme to work.
+
+</div>
+</div>
+</div>
+
+2. Use the `en.json` file to prepare your language file. Copy over the necessary key-value pairs and update the values.
+
+```json
+ ar.json                                           fr.json                                 
```

Comment:
Maybe it will be good to use this key `welcome_back` as an example for the whole tutorial. We can show what phrase will be rendered for Arabic version and what for English. 🍹

Reply:
I wasn't able to fine the appropriate file that contains welcome_back. I am happy to change it if you tell me what file it is in.